### DISCUSSION TITLE
Possibility to write logs in standard output / error

### DISCUSSION THREAD

**Discussion by marcobaro1991** [woocommerce/woocommerce#51070] [General]
Currently; in my production setup I send wordpress logs to stdout/stderr; Once sent I capture them and send them to external applications used for logs/metrics.
I should do the same thing with woocommerce.

It would be very useful to add an option whereby woocommerce no longer writes the logs to a particular file but uses native php functions to send in std output/error the various logs.

**Comment by moon0326** [comment]
Hi @marcobaro1991 

Good suggestion 👍 

In the meantime, you can use [woocommerce_logger_log_message](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/class-wc-logger.php#L186) filter to send the error to a custom destination.

**Reply by marcobaro1991** [reply]
hi @moon0326 
thx for your feedback :)

I was already aware of that filter :)
The idea is just to add the native option in woocommerce settings without going to use the filter in my custom theme/plugin

**Comment by coreymckrill** [comment]
> The idea is just to add the native option in woocommerce settings without going to use the filter in my custom theme/plugin

Converting this to an enhancement issue.
